plugins/lang-plan-a/src/main/java/org/elasticsearch/plan/a/PlanALexer.g4

```diff
@@ -0,0 +1,120 @@
+/*
```

Comment:
By convention, non Java code should not go to src/main/java but to src/main/resources.